data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,804 @@
+		"firing energy" 3
+		"infrared tracking" 0.9
+		"missile strength" 9
+		"submunition" "Nettle Active"
+		"stream"
+	description "Nettle missiles are minimalist kinetic kill vehicles designed for use in situations where nuclear weapons are considered unnecessary. Little more than a payload of tungsten rods, a thruster package, and a nose mounted sensor array, they can be manufactured in bulk at minimal cost."
+	description "By splitting into a number of tungsten rods upon approaching a target, only a fraction of the payload actually impacts it, but the damage is also distributed over a large area of effect, making evasion difficult."
+	description "This large Vertical Launch Cell array is made up of forty cells, each with Nettles stacked six missiles deep. Designed for rapid vertical cold-launch, it needs an open space comparable to a turret mount to function properly."
+
+
+outfit "Nettle Active"
+	weapon
+		sprite "projectile/shardactive"
+			"no repeat"
+			"frame rate" .25
+		"hit effect" "missile hit"
+		"live effect" "nettle exhaust" 1800
+		"die effect" "missile death"
+		"lifetime" 1800
+		"split range" 240
+		"acceleration" 0.3
+		"drag" .009
+		"turn" 9
+		"homing" 4
+		"infrared tracking" 0.9
+		"hit force" 300
+		"missile strength" 12
+		"submunition" "Fleck Flack" 30
+
+
+outfit "Fleck Flack"
+	weapon
+		sprite "projectile/fleck"
+			"frame rate" 30
+			"no repeat"
+		"die effect" "missile death"
+		"hit effect" "bombardment impact"
+		"inaccuracy" 18
+			narrow
+		"lifetime" 60
+		"random lifetime" 30
+		"random velocity" -6
+		"shield damage" 6
+		"hull damage" 9
+		"hit force" 90
+		"missile strength" 3
+
+effect "nettle exhaust"
+	sprite "effect/avgi flare/rcs"
+		"random start frame"
+		"scale" 0.5
+	"lifetime" 0
+
+
+outfit `"Orchid" Nuclear Missile`
+	category "Ammunition"
+	licenses
+		"Avgi Atomics"
+	cost 660000
+	thumbnail "outfit/orchid missile"
+	"mass" 0.9
+	"orchid capacity" -1
+	description "Unlike humanity, the Avgi do not have a strong taboo against the use of nuclear weapons in space. To them, nuclear weapons are simply another tool, albeit one with a particularly high energy density, to be used when the occasion calls for it. Nuclear weapons have been the Avgi's trump card against the Aberrant, the one weapon they have capable of reliably breaking their advanced shields and atomizing their armor."
+	description "This particular missile carries a compact thermonuclear warhead, a geometrically complex staged device with a far higher yield density than anything humanity has developed. It seems to be the product of centuries of development of nuclear arms, as opposed to humanity's long abstentation from nuclear weapons research."
+
+
+outfit "Orchid Magazine"
+	category "Ammunition"
+	licenses
+		"Avgi Atomics"
+	cost 1440000
+	thumbnail "outfit/unknown"
+	"mass" 1.2
+	"outfit space" -12
+	"orchid capacity" 12
+	ammo `"Orchid" Nuclear Missile`
+	description "Orchid storage"
+
+
+outfit "Orchid Launcher"
```

Comment:
These are too strong. I get that they're nukes, but 9,000 shield DPS from an 18 ton launcher, with a total of 108,000 damage in its ammo reserves, is just not balanced.
Even beyond that, I'm not sure what the counterplay is against these as missiles. They've got two tracking types, excellent homing, high speed (just over triple the top speed of firelights, which are themselves extremely fast), fast turning, very strong missile strength, and also an effective range of 36,000.
As far as I can tell, there's no good way to avoid the damage from these, and not much way to survive it either.

Reply:
Those all sound like much more sensible ideas, yeah.